js/viz/chart.d.ts

```diff
@@ -1945,12 +1955,12 @@ export interface dxChartCommonAxisSettings {
      * @public
      */
     width?: number;
-}
+};
 /**
- * @docid
- * @namespace DevExpress.viz
+ * @public
+ * @docid dxChartCommonAxisSettingsConstantLineStyle
```

Comment:
I think we can skip it here
`@type object` is not something meaningful, we have to use it in some places to correctly display the API Reference. 
In this particular place we can skip it because API Reference should be OK without it